test/blackbox-tests/test-cases/exec-watch/exec-watch-multi-levels.t/run.t

```diff
@@ -0,0 +1,34 @@
+Test the behaviour of exec watch mode when the current directory is not the
```

Comment:
This is a repro for this issue, right? can you land it first in its own PR?

Reply:
Made a new pr here: https://github.com/ocaml/dune/pull/10391